app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -430,7 +430,7 @@ public void onPageFinished(WebView view, String url) {
                 pageFragmentLoadState.onPageFinished();
                 updateProgressBar(false, true, 0);
                 webView.setVisibility(View.VISIBLE);
-                bridge.execute(JavaScriptActionHandler.setUp(leadImagesHandler.getPaddingTop()));
+                bridge.execute(JavaScriptActionHandler.setTopMargin(leadImagesHandler.getPaddingTop()));
```

Comment:
We shouldn't need to issue another call to `setTopMargin` here.  We should instead be setting it correctly in the initial `setup` call.